### PR TITLE
Removing word Bitcoin from forks

### PR DESCRIPTION
**Pull request by rockstardev**
We [got word](https://www.reddit.com/r/btc/comments/k3ya7k/the_people_behind_btcpay_server_are_a_bunch_of/) that we're inconsistently treating Bitcoin forks, so I've fixed the naming of methods and variables.

### CODE REVIEW

**Review by Kukks — APPROVED** [review]

**Comment by kristapsk** [comment]
Concept ACK

**Review by bolatovumar — APPROVED** [review]

File: BTCPayServer.Common/Altcoins/BTCPayNetworkProvider.BPlus.cs [hunk 1]
```diff
@@ -5,22 +5,22 @@ namespace BTCPayServer
...
             var nbxplorerNetwork = NBXplorerNetworkProvider.GetFromCryptoCode("XBC");
             Add(new BTCPayNetwork()
             {
                 CryptoCode = nbxplorerNetwork.CryptoCode,
-                DisplayName = "Bitcoinplus",
+                DisplayName = "BPlus",
                 BlockExplorerLink = NetworkType == NetworkType.Mainnet ? "https://chainz.cryptoid.info/xbc/tx.dws?{0}" : "https://chainz.cryptoid.info/xbc/tx.dws?{0}",
                 NBXplorerNetwork = nbxplorerNetwork,
-                UriScheme = "bitcoinplus",
+                UriScheme = "bplus-fix-it",
```
Comment (Kukks) [inline BTCPayServer.Common/Altcoins/BTCPayNetworkProvider.BPlus.cs:17]:
```suggestion
                UriScheme = "bitcoinplus",
```

just their uri scheme, let it be

**Review by Kukks — COMMENTED** [review]
leave the uri scheme as is so we don't break any usage

File: BTCPayServer.Common/Altcoins/BTCPayNetworkProvider.BPlus.cs (hunk 1 above)
Comment (Kukks) [reply BTCPayServer.Common/Altcoins/BTCPayNetworkProvider.BPlus.cs:17]: Apparently it is not even registered and used, so resolving this

**Pull request merged**